src/main/java/org/imsglobal/caliper/events/ContentAliasEvent.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * This file is part of IMS Caliper Analytics™ and is licensed to
+ * IMS Global Learning Consortium, Inc. (http://www.imsglobal.org)
+ * under one or more contributor license agreements.  See the NOTICE
+ * file distributed with this work for additional information.
+ *
+ * IMS Caliper is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation, version 3 of the License.
+ *
+ * IMS Caliper is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+ * A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.imsglobal.caliper.events;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.imsglobal.caliper.actions.Action;
+import org.imsglobal.caliper.validators.EventValidator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@SupportedActions({
+        Action.ASSIGNED_ALIAS
```

Comment:
I think we can re-use the [ContentEvent.java](https://github.com/wn-doolittle/caliper-java/blob/117e1a5617c27b90cd12d719c4089a896479f7c2/src/main/java/org/imsglobal/caliper/events/ContentEvent.java) class, we just need to add this action there.
For reference you can look at `HierarchyEvent.java` and `PathEvent.java`